src/linalg/svd.rs

```diff
@@ -628,15 +737,15 @@ where
     /// Computes the singular values of this matrix.
     #[must_use]
     pub fn singular_values(&self) -> OVector<T::RealField, DimMinimum<R, C>> {
-        SVD::new(self.clone_owned(), false, false).singular_values
+        SVD::new_unordered(self.clone_owned(), false, false).singular_values
```

Comment:
Since `fn svd(...)` returns an ordered result, it would be more coherent to have `fn singular_values(...)` return an ordered result too. And add `fn singular_values_unordered(...)`.

Reply:
FIne with me! The (possible breaking) additional constraints on the allocator are not a problem, are they?

Reply:
Oh yes, I forgot that only the "_unordered" function have the same constraints... Never mind :)